libsplinter/Cargo.toml

```diff
@@ -111,9 +112,9 @@ biome-credentials = ["biome", "biome-user", "database", "bcrypt"]
 biome-key-management = ["biome", "database"]
 biome-notifications = ["biome", "database"]
 biome-user = ["biome", "database"]
-circuit-read = []
+circuit-read = ["rest-api", "rest-api-actix"]
```

Comment:
rest-api-actix should not be automatically pulled in; instead, both circuit-read and rest-api-actix should be required to turn on these implementations.

Reply:
Fixed